src/tool_manager.ts

```diff
@@ -259,6 +260,20 @@ class ToolManager {
     return r_resolve;
   }
 
+  private escapeCmd(cmd: string) {
+      // Double quote command (should work in at least cmd.exe and bash)
+      let cmdEsc = `"${cmd}"`;
+
+      // Fetch Windows shell type
+      let shell = vscode.workspace.getConfiguration("terminal.integrated.shell").get("windows", "");
+
+      // For powershell we prepend an & to prevent the command being treated as a string
+      if (shell.endsWith("powershell.exe") && process.platform === "win32") {
```

Comment:
Does 64bit of the VSCode is also valid for `win32` ?

Reply:
Yep, Node.js returns `win32` regardless.